addons/pos_hr/tests/test_frontend.py

```diff
@@ -0,0 +1,68 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
```

Comment:
@mart-e Do you think those tests are needed? it seems quite big just to check that a button is in the view

Reply:
probably not useful to check if there is a close button but checking the pos_hr features works is not a bad idea